.gitignore

```diff
@@ -114,3 +114,5 @@ dmypy.json
 # Pyre type checker
 .pyre/
 
+# check migrations
```

Comment:
Here executing `flask db init` was not happy having this folder pushed here. So better remove it completely from repo since it does not have info. However in real projects ur right u would have a folder like that here for migrations to execute, but you would not have to do `db init` I guess. 
The README should be my entry point to your software and if possible as easy as possible.